README.md

```diff
@@ -149,9 +149,18 @@ Notes for developers
 
 To start contributing to Mosh, install the following dependencies:
 
+Debian, Windows Subsystem for Linux:
+
+```
+$ sudo apt install -y build-essential debhelper-compat protobuf-compiler \
+    libprotobuf-dev pkg-config libutempter-dev zlib1g-dev libncurses5-dev \
+    libssl-dev bash-completion locales tmux less
```

Comment:
You shouldn't need locales (and we should remove that from the debian/rules file)

Reply:
Done